straxen/plugins/online_monitor.py

```diff
@@ -49,6 +49,18 @@
              '"near_s1_hists", cuts should be separated using "&"'
              'For example: (tight_coincidence > 2) & (area_fraction_top < 0.1)'
              'Default is no selection (other than "area_vs_width_min_gap")'),
+    strax.Option(
+        'lone_hits_cut_string',
+        type=str,
+        default='(area > 0.2) & (area < 2.0)',
+        help='Selection (like selection_str) applied to data for '
+             '"lone-hits", cuts should be separated using "&")'),
+    strax.Option(
+        'lone_hits_min_gap',
+        type=int,
+        default=15_000,

```

Comment:
I didn't know that strax can identify 15_000 as a number. nice!

Reply:
:D The credit goes to python though. It is a vanilla feature.